packages/pages/src/common/src/processEnvVariables.ts

```diff
@@ -0,0 +1,23 @@
+import { loadEnv } from "vite";
+
+/**
+ * Produces a map of environment variables filtered to those relevant
+ * to the vite application, so environment variables can be accessed
+ * without a runtime specific wrapping object.
+ *
+ * @param prefix string specifying the beginning of the keys to match
+ * @returns
+ */
+export const processEnvVariables = (
+  prefix = "VITE"
```

Comment:
To clarify, looks like processEnvVariables is called with `projectStructure.envVarPrefix` everythime and I see that the default when constructing ProjectStructure instance have `envVarPrefix: "YEXT_PUBLIC"`, is that what the spec expects? or does it also want envs with "VITE" prefix to pass through

Reply:
If `projectStructure.envVarPrefix` was undefined the prefix would default to "VITE" so I made that the default value here as well.